src/app/DTT/radio/index.js

```diff
@@ -99,9 +99,9 @@ const Write = ({
   return (
     <Stack
       ml={1}
-      direction={outerStackVertical ? 'row' : 'column'} // just making sure that longer sets of options don't end up weirdly arranged
-      spacing={outerStackVertical ? 0 : 1}
-      justifyContent={outerStackVertical ? 'space-between' : 'flex-start'}
+      direction={outerStackVertical || !boolean ? 'row' : 'column'} // just making sure that longer sets of options don't end up weirdly arranged
```

Comment:
rename boolean to something more meaningful and don't use the name of data types already provided by the language.

Reply:
Renamed `boolean` to `isBooleanSelect`